Documentation/guides/BuildProcess.md

```diff
@@ -1173,6 +1173,29 @@ To use an environment variable to store your password you can do the following
 </PropertyGroup>
 ```
 
+-   **JarsignerTimestampAuthorityUrl** &ndash; This property
+    allows you to specify a Url to a timestamp authority
+    service. This can be used to make sure your apk signature
+    includes a timestamp.
+    See [time-of-signing](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) for more details.
+
+```xml
+<PropertyGroup>
+    <JarsignerTimestampAuthorityUrl >http://example.tsa.url</JarsignerTimestampAuthorityUrl>
+</PropertyGroup>
+```
+
+-   **JarsignerTimestampAuthorityCertificateAlias** &ndash; This 
+    property allows you to specify an alias in the keystore
+    for a timestamp authority.
+    See [time-of-signing](https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html) for more details.
+
+```xml
+<PropertyGroup>
+    <JarsignerTimestampAuthorityCertificateAlias >Alias</JarsignerTimestampAuthorityCertificateAlias>
+</PropertyGroup>
+```
+
```

Comment:
@brendanzagaeski: I'd like a second opinion, but should these new bullet points go *before* the paragraphs starting with:

```md
For example, consider the following `keytool` invocation:
```

*After* the `AndroidSigningStorePass` bullet point?

I also think that the xml code blocks should be indented, as with the `AndroidSigningStorePass` bullet point.

Reply:
Yes, I think it makes sense to move these new bullet points to be part of the existing list of properties so all the properties are in one contiguous list. Admittedly, the examples given after ``For example, consider the following `keytool` invocation:`` don't mention these new properties, but I think that's fine too.

And yes, I think that in this case it would work nicely to indent the example XML blocks so that they're part of the list items. The example XML blocks are directly matched to the individual properties, so it sounds good to me to include them within the list items.